README.md

```diff
@@ -91,27 +94,75 @@ Run `npm test` or `yarn test` in the console.
 * [Query-string](https://www.npmjs.com/package/query-string) - Used to retrieve the variable values in the HTTP query string.
 * [Eslint](https://eslint.org/) - A tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
 
+## Versioning
+
+This is version 1.0 (Release canvas 1)
```

Comment:
I would personally not explicitly mention the release canvas but to merely have it be evident in the version number. I would opt for `V 2.0.1` (reflecting that we're using someone's existing codebase)

Reply:
Completed